src/components/NavBar/RecentlySearchedAssets.ts

```diff
@@ -86,7 +86,15 @@ export function useRecentlySearchedAssets() {
     shortenedHistory.forEach((asset) => {
       if (asset.address === 'NATIVE') {
         // Handles special case where wMATIC data needs to be used for MATIC
-        const native = nativeOnChain(CHAIN_NAME_TO_CHAIN_ID[asset.chain] ?? SupportedChainId.MAINNET)
+        const chain = supportedChainIdFromGQLChain(asset.chain)
+        if (!chain) {
+          Sentry.withScope((scope) => {
```

Comment:
maybe create a helper function that does all the Sentry stuff for you across the app? otherwise feels a bit repetitive

```
function assertDefined(chain, { extras, errorMessage }) {
  Sentry.withScope((scope) => {
    Object.entries(extra).map(([k, v]) => {
      scope.setExtra(k, v)
    })
    Sentry.captureException(new Error(errorMessage))
  })
}
```

Reply:
made some slight modifications wrt naming and removed chain as an arg as it was redundant with the extras